assets/js/src/core/modules/data-object/draft/hooks/use-draft-data.ts

```diff
@@ -43,7 +43,6 @@ export const useDraftDataReducers = (entityAdapter: EntityAdapter<DataObjectDraf
   const modifyDraft = (state: EntityState<DataObjectDraft, number>, id: number, modification: (draft: DataObjectDraft) => DataObjectDraft): void => {
     const draft = entityAdapter.getSelectors().selectById(state, id)
     if (draft === undefined) {
-      console.error(`Data object draft with id ${id} not found`)
```

Comment:
Is there a reason why we remove all those console.error() logs?

Reply:
Yea, i removed it because it would show an error if thers no draft (so if the dataobject is not open). IMHO thats not an error but just a "do nothing if thats the case"